dspace-rest/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -30,11 +30,13 @@
     </bean>
 
     <!-- Inject the SolrLoggerUsageEventListener into the EventService  -->
+    <!-- Comment out to enable service in DSpace 7 
```

Comment:
Tiny thing, but I'd recommend us leaving ourselves a more useful note here, especially now that you have a ticket created for this issue. Maybe something like:

"Temporarily commented out to get REST API working on master.  This should be uncommented again once DS-3815 is resolved"